enterprise_gateway/services/processproxies/processproxy.py

```diff
@@ -15,13 +15,16 @@
 import getpass
 from tornado import web
 import subprocess
+import base64
 from ipython_genutils.py3compat import with_metaclass
 from socket import *
 from jupyter_client import launch_kernel, localinterfaces
 from calendar import timegm
 from notebook import _tz
 from zmq.ssh import tunnel
 from enum import Enum
+from Crypto.Cipher import AES
+
```

Comment:
Looks like both new import references should get reflected in `requirements.txt` to resolve the build issues.